ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+-  `Create a service account key/secret pair`_
+-  `Run a Java producer: before and after ACLs`_
+-  `Run a Java producer: showcase a Prefix ACL`_
+-  `Run |kconnect| and kafka-connect-datagen connector with permissions`_
+-  `Run a Java consumer: showcase a Wildcard ACL`_
+-  `Delete the API key, service account, Kafka topics, Kafka cluster, environment,
+   and the log files`_
+
+
+Create a new environment and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a new |ccloud| environment
+   ``demo-script-env``:
+
+   .. code-block:: bash
+
+      ccloud environment create demo-script-env -o json
+
+   The output is in JSON format:
+
+   .. code-block:: text
+
+      {
+        "id": "env-5qz2q",
+        "name": "demo-script-env"
+      }
+
+
+#. Specify ``env-5qz2q`` as the active environment by running the following
+   command:
+
+   .. code-block:: bash
+
+       ccloud environment use env-5qz2q
+
+   You should see the following message: ``Now using "env-5qz2q" as the default (active) environment.``
+
+
+Create a new Kafka cluster and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a new |ccloud| cluster
+   ``demo-kafka-cluster``:
+
+   .. code-block:: bash
+
+      ccloud kafka cluster create demo-kafka-cluster --cloud aws --region us-west-2
+
+   .. note::
+
+      It may take up to 5 minutes for the |ak| cluster to be ready.
+
+#. Verify you see the following output:
+
+   .. code-block:: text
+
+      +--------------+---------------------------------------------------------+
+      | Id           | lkc-x6m01                                               |
+      | Name         | demo-kafka-cluster                                      |
+      | Type         | BASIC                                                   |
+      | Ingress      |                                                     100 |
+      | Egress       |                                                     100 |
+      | Storage      |                                                    5000 |
+      | Provider     | aws                                                     |
+      | Availability | LOW                                                     |
+      | Region       | us-west-2                                               |
+      | Status       | UP                                                      |
+      | Endpoint     | SASL_SSL://pkc-4kgmg.us-west-2.aws.confluent.cloud:9092 |
+      | ApiEndpoint  | https://pkac-ldgj1.us-west-2.aws.confluent.cloud        |
+      +--------------+---------------------------------------------------------+
+
+#. Specify ``lkc-x6m01`` as the active |ak|| cluster by running the following
+   command:
+
+   .. code-block:: bash
+
+      ccloud kafka cluster use lkc-x6m01
+
+   Running the previous command sets the |ak| cluster ``lkc-x6m01`` as the active
```

Comment:
Can this statement be integrated into the step text itself?

Reply:
Done